scripts/msys2-install-dependencies.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#pacman --noconfirm --ask 20 -Sy
+#pacman --noconfirm --ask 20 -Su
```

Comment:
What are these commented-out lines?

Reply:
That is for core / system update. I'm still not sure if it's required. At some point it seemed to have broken the installed cmake but it might just be all those installations in path breaking things.